packages/addon-dev/src/rollup-hbs-plugin.ts

```diff
@@ -12,6 +12,10 @@ export default function rollupHbsPlugin({
   return {
     name: 'rollup-hbs-plugin',
     async resolveId(source: string, importer: string | undefined, options) {
+      let hbsFilename = source.replace(/\.\w{1,3}$/, '') + '.hbs';
+      if (hbsFilename !== source) {
+        this.addWatchFile(hbsFilename);
```

Comment:
@ef4 I had to add addWatchFile to all 3 methods to get this to work correctly on windows